crates/composer/src/main.rs

```diff
@@ -1,10 +1,9 @@
-use composer_api::Event;
-use composer_api::DEFAULT_SERVER_ADDRESS;
+#![warn(clippy::all, clippy::clone_on_ref_ptr)]
+
+use composer_api::{Event, DEFAULT_SERVER_ADDRESS};
 use eyre::Result;
 use rodio::{source::Source, Decoder, OutputStream, OutputStreamHandle};
-use std::fs::File;
-use std::io::BufReader;
-use std::net::UdpSocket;
+use std::{fs::File, io::BufReader, net::UdpSocket};
```

Comment:
Good! I was hoping `fmt` would enforce a single, canonical way of grouping and ordering imports.

Reply:
Though personally, I'd use `StdExternalCrate` rather than `One`. .)